docs/gallery/general/plot_configurator.py

```diff
@@ -0,0 +1,112 @@
+"""
+How to Configure Term Validations
+=================================
+
+This is a user guide for how to curate and take advantage of configuration files in
+order to more easily validate terms within datasets or attributes.
+
+Introduction
+-------------
+Users will create a configuration YAML file that outlines the fields (within a neurodata type)
+they want to be validated against a set of allowed terms. 
+After creating the configuration file, users will need to load the
+configuration file with the :py:func:`~pynwb.load_type_config` method.
+With the configuration loaded, every instance of the neurodata
+types defined in the configuration file will have the respective fields wrapped with a
+:py:class:`~hdmf.term_set.TermSetWrapper`.
+This automatic wrapping is what provides the term validation for the field value.
+For greater control on which datasets and attributes are validated
+against which sets of allowed terms, use the 
+:py:class:`~hdmf.term_set.TermSetWrapper` on individual datasets and attributes instead.
+You can follow the 
+`TermSet tutorial in the HDMF documentation 
+<https://hdmf.readthedocs.io/en/stable/tutorials/plot_term_set.html#sphx-glr-tutorials-plot-term-set-py>`_
+for more information.
+
+To unload a configuration, simply call :py:func:`~pynwb.unload_type_config`.
+We also provide a helper method to see the configuration that has been loaded:
+:py:func:`~pynwb.get_loaded_type_config`
+
+
+How to make a Configuration File
+--------------------------------
+To see an example of a configuration file, please refer to ``nwb_config.yaml`` under ``src/config``.
+The configuration file uses YAML syntax. The
+user will construct a series of nested dictionaries to encompass all the necessary information.
```

Comment:
@mavaylon1 The writing style here is very formal, e.g., saying "the user will". Could you please adjust the writing style to speak to the user directly, as if you were guiding them through this one on one? Second-person, imperative voice. e.g., "To accomplish X, do Y", or "Next, let's do X", and sometimes "You can do X", or "To accomplish X, we will do Y"? We don't have a style guide for this, but I think generally writing as if you were speaking directly to the user is more approachable and friendly for them.

Reply:
I don't disagree, but also I don't want to rewrite this currently. I also checked my other tutorials and they also have my style that would also need to change. I can do a tutorial rewrite as a low issue ticket if that is a middle ground option.

Reply:
OK that's fine. Please create a ticket so we don't forget to do this eventually.